Dockerfile

```diff
@@ -16,22 +14,19 @@ RUN set -e \
     && tar -C tmp_install -czf /postgres_install.tar.gz .
 
 # Build zenith binaries
-#
-#FROM zimg/rust:1.58 AS build
-FROM zenithdb/build:buster-20220414 AS build
+FROM zimg/rust:1.58 AS build
 ARG GIT_VERSION=local
 
 ARG CACHEPOT_BUCKET=zenith-rust-cachepot
 ARG AWS_ACCESS_KEY_ID
 ARG AWS_SECRET_ACCESS_KEY
-ENV RUSTC_WRAPPER /usr/local/cargo/bin/cachepot
 
 COPY --from=pg-build /pg/tmp_install/include/postgresql/server tmp_install/include/postgresql/server
 COPY . .
 
 # Show build caching stats to check if it was used in the end.
 # Has to be the part of the same RUN since cachepot daemon is killed in the end of this RUN, loosing the compilation stats.
-RUN cargo build --release && /usr/local/cargo/bin/cachepot -s
+RUN cargo build --release && cachepot -s
 
 # Build final image
 #
```

Comment:
huh, seems that the last layer was `bullseye` anyway and this image is used in e2e test. So maybe there was no problem with older glibc?)

Reply:
According to https://github.com/neondatabase/neon/pull/1347 it was though and that line did not change since then?

Maybe @cicdteam knows more.
But looks like we are not breaking it more in terms of versions then.